### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by luckloa**
Submit my code

### CODE REVIEW

**Review by Wthongkom — CHANGES_REQUESTED** [review]
Please change your code

File: src/test/java/HelloThere/YOLO.java [hunk 1]
```diff
@@ -5,6 +5,7 @@ public class YOLO {
 	public static void main(String[] args) {
 
 		System.out.println("Hello");
+		System.out.println("hihihi");
```
Comment (Wthongkom) [inline src/test/java/HelloThere/YOLO.java]: Add another print statement

Comment (Wthongkom) [reply src/test/java/HelloThere/YOLO.java:8]: dsss

Comment (luckloa) [reply src/test/java/HelloThere/YOLO.java]: I will fix it

**Pull request merged**